backup.ps1

```diff
@@ -334,7 +336,15 @@ function Send-Email {
         # create a temporary error log to log errors; can't write to the same file that Send-MailMessage is reading
         $temp_error_log = $ErrorLog + "_temp"
 
-        Send-MailMessage @ResticEmailConfig -From $ResticEmailFrom -To $ResticEmailTo -Credential $credentials -Subject $subject -Body $body @attachments 3>&1 2>> $temp_error_log
+        #sender ([MimeKit.MailboxAddress] http://www.mimekit.net/docs/html/T_MimeKit_MailboxAddress.htm, required)
+        $From = [MimeKit.MailboxAddress]$ResticEmailFrom;
+        $RecipientList = [MimeKit.InternetAddressList]::new();
+        $RecipientList.Add([MimeKit.InternetAddress]$ResticEmailTo);
+        $Port = 465
+        #authentication ([System.Management.Automation.PSCredential], optional)
+        $Credential = [System.Management.Automation.PSCredential]::new($ResticEmailFrom, (ConvertTo-SecureString -String $ResticEmailPassword -AsPlainText -Force));
+#send message
+        Send-MailKitMessage -SMTPServer $PSEmailServer -Port $Port -Credential $Credential -From $From -RecipientList $RecipientList -Subject $subject -TextBody $body 3>&1 2>> $temp_error_log    
```

Comment:
Apart from the aftermentioned change of the port variable, I reckon `-UseSecureConnectionIfAvailable` is needed more often than not and, if it works the way I think it does, it won't affect connections in which a secure connection is not available. So this would become:

`Send-MailKitMessage -SMTPServer $PSEmailServer -Port $PSEmailPort -UseSecureConnectionIfAvailable -Credential $Credential -From $From -RecipientList $RecipientList -Subject $subject -TextBody $body 3>&1 2>> $temp_error_log`